src/routes/transactions/mappers/common/native-staking.mapper.ts

```diff
@@ -347,6 +388,9 @@ export class NativeStakingMapper {
    * Gets the net value (staked + rewards) to withdraw from the native staking deployment
    * based on the length of the publicKeys field in the transaction data.
    *
+   * Note: this can only be used with `validatorsExit` or `batchWithdrawCLFee` transactions
+   * as the have `_publicKeys` field in the decoded data.
```

Comment:
We should reconsider all the `*FromDataDecoded` methods as they do not work for `deposit` calls. We can get the respective value by means of decoding the transaction data directly.